scripts/post_plot.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -l
+
+echo "post_plot_fcst.sh starts"
+echo "CDATE is " $CDATE
+cd $RUNCDATE
+echo "Location of experiment: "`pwd` 
+
+IceAnlDir=$RUNCDATE/Data
+OceanFcstDir=$RUNCDATE/fcst
+FiguresDir=$RUNCDATE/Figures
+
+[ ! -d $FiguresDir ] && mkdir -p $FiguresDir
+
+echo "Figures are created at $FigureDir"
+
+GridFile=$(find $OceanFcstDir -name 'ocn_*.nc' -print -quit)
+if [[ -z "$GridFile" ]] ; then
+    echo "Grid file containing geolon/geolat does not exist ..."
+    exit 1
+fi
+
+# 1. Plot Ice analysis
+files2plot=( "$IceAnlDir"/cic.socagodas.an.*.nc )
+if [ -e "${files2plot[0]}" ]; then
+    cp -rf $IceAnlDir/cic.socagodas.an.*.nc $OceanFcstDir
+    python $SOCA_EXEC/ice.plot.py                    \
```

Comment:
Try "ipython -- " instead of python, Hopefully, Errno101 will not occur and we can activate the set -x again.